lib/curator/ueberauth/ueberauth.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Curator.Ueberauth do
+  use Curator.Extension
+
+  def unauthenticated_routes do
+    quote do
+      get "/:provider", UeberauthController, :request
```

Comment:
I think we might have to make this more configurable.. Their auth controller might have a different name. Maybe `curator_ueberauth` config could have a key for `auth_controller` and `action`?